modules/virt-measuring-latency-vm-secondary-network.adoc

```diff
@@ -0,0 +1,252 @@
+
+:_content-type: PROCEDURE
+[id="virt-measuring-latency-vm-secondary-network_{context}"]
+= Checking network connectivity and latency for virtual machines on a secondary network
+
+As a cluster administrator, you use a predefined checkup to verify network connectivity and measure latency between virtual machines (VMs) that are attached to a secondary network interface. Set up the namespace and service account for the framework; create the `ClusterRole` and `ClusterRoleBinding` objects for the service account; enable permissions for the checkup; create the input config map; and run the checkup. After you record the results of the checkup, delete all the framework resources that you previously created.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You logged in to the cluster as a user with the `cluster-admin` role.
+* The cluster has at least two worker nodes.
+* The Multus Container Network Interface (CNI) plug-in is installed on the cluster.
+* You configured a network attachment definition for a namespace.
+
+.Procedure
+
+. Create a configuration file that contains the resources to set up the framework. This includes a namespace and service account for the framework, and the `ClusterRole` and `ClusterRoleBinding` objects to define permissions for the service account.
++
+.Example framework manifest file
+[%collapsible]
+====
+[source,yaml]
+----
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kiagnose
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: kiagnose
+  namespace: kiagnose
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: kiagnose
+rules:
+  - apiGroups: [ "" ]
+    resources: [ "configmaps" ]
+    verbs:
+      - get
+      - list
+      - create
+      - update
+      - patch
+  - apiGroups: [ "" ]
+    resources: [ "namespaces" ]
+    verbs:
+      - get
+      - list
+      - create
+      - delete
+      - watch
+  - apiGroups: [ "" ]
+    resources: [ "serviceaccounts" ]
+    verbs:
+      - get
+      - list
+      - create
+  - apiGroups: [ "rbac.authorization.k8s.io" ]
+    resources:
+      - roles
+      - rolebindings
+      - clusterrolebindings
+    verbs:
+      - get
+      - list
+      - create
+      - delete
+  - apiGroups: [ "rbac.authorization.k8s.io" ]
+    resources:
+      - clusterroles
+    verbs:
+      - get
+      - list
+      - create
+      - bind
+  - apiGroups: [ "batch" ]
+    resources: [ "jobs" ]
+    verbs:
+      - get
+      - list
+      - create
+      - delete
+      - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: kiagnose
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kiagnose
+subjects:
+  - kind: ServiceAccount
+    name: kiagnose
+    namespace: kiagnose
+...
+----
+====
+
+. Apply the framework manifest:
++
+[source,terminal]
+----
+$ oc apply -f <framework-manifest>.yaml
```

Comment:
For any file name examples like this, make sure they use underscores instead of hyphens. I might also just name them all <file_name>.yaml, but I am boring. :)